### ISSUE TITLE
Access to sitelinks & parsing P1082

### ISSUE THREAD

**Issue by lenalebt** [syvb/wikidata#2]
Hey!
I'm currently playing around with https://github.com/p-e-w/wd2sql which is using this library, and I'm trying to extract site links as well. If I'm not mistaken, that's completely unimplemented here, right? Sorry for asking dumb questions maybe, I just today started with Rust, so maybe I'm just overlooking something :sweat_smile: !?

I also had some trouble accessing [P1082](https://www.wikidata.org/wiki/Property:P1082) claims, which seem to have an incompatible format somehow; workaround for now is to ignore those claims

Would you be willing to release a new version if I make changes & provide a PR?

Cheers,
Lena

**Comment by syvb** [comment]
Sitelinks are indeed unsupported, it would be great to add support for them. I would release a new version if you made a PR.

I just noticed that there are some compiler warnings when building from source, so I just pushed some commits to fix them.

Re P1082 claims, I did some testing and they seem to be parsed fine at least in some cases - e.g. [here is how Q40168 is parsed](https://gist.github.com/syvb/fd3608b5148418e77c1baed2a49a5ab8). What item is causing parsing problems?
